docs/transports.md

```diff
@@ -116,11 +111,27 @@ You can see an example using a async iterator with ESM:
 
 ```js
 import build from 'pino-abstract-transport'
+import SonicBoom from 'sonic-boom'
+import { once } from 'events'
 
 export default async function (opts) {
+  // SonicBoom is necessary to avoid loops with the main thread.
+  // It is the same of pino.destination().
+  const destination = new SonicBoom({ dest: opts.destination || 1, sync: false })
```

Comment:
Why don't we show using `pino.destination` here instead?

Reply:
If I am writing a module to be published on npm I do not want to depend on pino. This should probably be written somewhere.

Reply:
That makes sense. I think these docs as written, though, are from the perspective of someone writing an _application_ that depends on Pino. Thus, they will already have the dependency.

Reply:
Should we mention both?

Reply:
Yeah, but we can follow up with another PR at a later date if you like.

Reply:
This page needs an overhaul I think :(. Let's land this and iterate.